src/Common/Doctrine/Entity/Meta.php

```diff
@@ -167,6 +167,23 @@ public function update(
         $this->seoIndex = $seoIndex;
     }
 
+    public static function copy(self $meta): self
```

Comment:
why don't you use the __clone function for this?

Reply:
Good one 5ab651c2